.travis.yml

```diff
@@ -118,6 +118,6 @@ script:
 
 after_success:
     # Upload to coveralls.
-  - if [ -f build/coverage.info ]; then
-      coveralls-lcov --repo-token ${COVERALLS_TOKEN} build/coverage.info;
+  - if [ -f coverage.info ]; then
```

Comment:
This doesn't look right considering that we build and run everything except for tck in the build dir.